### PR TITLE
Setup instructions for the flask server

### PR DESCRIPTION
**Pull request by carpecodeum**
# Description

These are the setup instructions and changes in the app.py needed to add the labellab-flask directory to the PYTHONPATH env variable.

Fixes #424 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

Also, include screenshots of app for the verification and reviewing purpose.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,3 +123,52 @@ or run the Flutter application.
...
+-   **[SQLAlchemy](https://www.sqlalchemy.org)** - An SQL ORM
+<br/>
+
+1. Make a database in your MYSQL 
+   ```bash
+   CREATE {DATABASE | SCHEMA} [IF NOT EXISTS] labellab
+   ```
+2. Create a folder called called venv using 
+   ```bash
+   cd labellab-flask && mkdir venv
```
Comment (Tushar19varshney) [inline README.md:146]: When we create virtualenv, venv automatically gets created. We don't need to manually create this directory.

File: README.md [hunk 2]
```diff
@@ -123,3 +123,52 @@ or run the Flutter application.
...
+```
+-   **[MySQL](https://www.mysql.com/)** - Preferrably MySQL-5.7 database is used. 
+-   **[SQLAlchemy](https://www.sqlalchemy.org)** - An SQL ORM
+<br/>
+
+1. Make a database in your MYSQL 
+   ```bash
+   CREATE {DATABASE | SCHEMA} [IF NOT EXISTS] labellab
+   ```
+2. Create a folder called called venv using
```
Comment (Tushar19varshney) [inline README.md:144]: Correct this typo error.

**Comment by Tushar19varshney** [comment]
@carpecodeum Just look at the comments above.

File: README.md (hunk 1 above)
Comment (carpecodeum) [reply README.md:146]: okay sure I will change this thanks for pointing this out :)

File: README.md (hunk 2 above)
Comment (carpecodeum) [reply README.md:144]: sure!

**Comment by Tushar19varshney** [comment]
@carpecodeum Looks good to me!

**Pull request merged**